devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -383,4 +534,20 @@ private logDebug(msg) {
 
 private logTrace(msg) {
     log.trace "$msg"
+}
+
+private isButtonAvailable() {
+    if(device == null){
+        log.error "isButtonAvailable device = null"
+        return true
+    }else{
+        log.debug "isButtonAvailable device.rawDescription = ${device.rawDescription}"
+        def v20 = "${device.rawDescription}".contains("model:EE01")
+        def v21 = "${device.rawDescription}".contains("model:EE03")
+        def v22 = "${device.rawDescription}".contains("model:A005")
+        def v23 = "${device.rawDescription}".contains("model:BB01")
+        def v24 = "${device.rawDescription}".contains("model:BB03")
+        def v2 = v20||v21||v22||v23||v24
+        return v2
```

Comment:
Is it really needed to declare variables here ?
You might just `return "${device.rawDescription}".contains("model:EE01") || "${device.rawDescription}".contains("model:EE03") || ... etc`